content/kotlin/concepts/Objects/Objects.md

```diff
@@ -0,0 +1,55 @@
+---
+Title: 'objects'
+Description: 'Kotlin object declaration is a special type of declaration that allows the creation of a single instance of a class, commonly known as a Singleton.'
```

Comment:
I don't think the issue for this entry was specific regarding what should and shouldn't be addressed about objects. But, I think this entry could take a broader approach and include the other ways in which objects are used, such as object expressions and companion objects. Please revise this entry to include the other functions of objects in the kotlin language.

Reply:
Understood. I'm going to expand the entry and add the other definitions (and examples) about Kotlin Objects. I hope I can do it tomorrow. Thank you so much @caupolicandiaz.